crates/torii/libp2p/src/server/mod.rs

```diff
@@ -291,6 +292,7 @@ impl<P: Provider + Sync> Relay<P> {
                                         peer_id = %peer_id,
                                         "Message set."
                                     );
+                                    self.db.execute().await.unwrap();
```

Comment:
We should also execute the queries for the set_entity that is further down. Can we refactor the logic here to reuse the same self.db.execute() everywhere? We can add a function that sets the entity and directrly executes the db queries